core/src/main/java/org/mobilitydata/gtfsvalidator/notice/NoticeContainer.java

```diff
@@ -36,6 +36,8 @@
 public class NoticeContainer {
   private static final Gson DEFAULT_GSON =
       new GsonBuilder().serializeNulls().serializeSpecialFloatingPointValues().create();
+  private static final Gson PRETTY_GSON =
```

Comment:
We risk to have those Gson objects out of sync. Let's try this: 

  private static final Gson PRETTY_GSON = new GsonBuilder(DEFAULT_GSON).setPrettyPrinting().create();

Reply:
Unfortunately `GsonBuilder(Gson gson)` is package-private and cannot be accessed from `NoticeContainer` class. What do you think of the following?
```java
  private static final Gson PRETTY_GSON = DEFAULT_GSON.newBuilder().setPrettyPrinting().create();
```

https://www.javadoc.io/doc/com.google.code.gson/gson/2.8.5/com/google/gson/Gson.html#newBuilder--